src/extension.ts

```diff
@@ -150,15 +151,15 @@ export async function activate(context): Promise<extensionapi.ExtensionAPI> {
         registerCommand('extension.vsKubernetesUseContext', useContextKubernetes),
         registerCommand('extension.vsKubernetesClusterInfo', clusterInfoKubernetes),
         registerCommand('extension.vsKubernetesDeleteContext', deleteContextKubernetes),
-        registerCommand('extension.vsKubernetesUseNamespace', (obj) => { useNamespaceKubernetes(kubectl, obj); } ),
+        registerCommand('extension.vsKubernetesUseNamespace', (explorerNode: explorer.KubernetesObject) => { useNamespaceKubernetes(kubectl, explorerNode); } ),
```

Comment:
I get this but I'd prefer it to be in a separate PR as it's not related to logs (and yeah I know I do this sometimes as well)... just makes it clearer when reading the diffs in six months' time.

Reply:
considering it's not a drastic functional change, and it's fairly small, it should be okay to leave it here.